Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -352,14 +352,14 @@ - (void)_installScreenTimeWebpageController
     if (!PAL::isScreenTimeFrameworkAvailable())
         return;
 
-    if (!_page->preferences().screenTimeEnabled())
+    if (_page && !_page->preferences().screenTimeEnabled())
         return;
 
     if (!_screenTimeWebpageController) {
         _screenTimeWebpageController = adoptNS([PAL::allocSTWebpageControllerInstance() init]);
-        [_screenTimeWebpageController addObserver:self forKeyPath:@"URLIsBlocked" options:(NSKeyValueObservingOptionOld | NSKeyValueObservingOptionNew) context:screenTimeWebpageControllerBlockedKVOContext];
+        [_screenTimeWebpageController addObserver:self forKeyPath:@"URLIsBlocked" options:(NSKeyValueObservingOptionOld | NSKeyValueObservingOptionNew) context:&screenTimeWebpageControllerBlockedKVOContext];
 
-#if PLATFORM(MAC)
+#if PLATFORM(MAC) || PLATFORM(IOS_FAMILY)
```

Comment:
Can remove this `#if` entirely, since this is just `PLATFORM(COCOA)`, and this file is already `PLATFORM(COCOA)`.